Documentation/ViewDesigner/Examples/VDTabbed.md

```diff
@@ -0,0 +1,381 @@
+## View Designer Tabbed - json
+
+To replicate the View Designer Tabbed Workbook, copy and paste the following code into the Advanced editor, denoted by the </> symbol in toolbar
+
+![Avanced Editor Toolbar](./ToolBar.png)
+
+Users may have to update their query settings and subscriptions to their own accessible resources
+
+[Return to Conversion Options](../ConversionOptions.md)
+
+```
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "crossComponentResources": [
+          "/subscriptions/1f3fa6d2-851c-4a91-9087-1a050f3a9c38/resourcegroups/defaultresourcegroup-eus/providers/microsoft.operationalinsights/workspaces/defaultworkspace-1f3fa6d2-851c-4a91-9087-1a050f3a9c38-eus"

```

Comment:
Should strip out the "crossComponentResources" which has your subscriptionId
This is actually a common problem for authors, to publish a template they must manually go through and strip these out.

Reply:
I wonder if we should make an "examples" or "samples" folder somewhere that doesn't get published as templates (maybe?) but has all the normal validations applied to them to catch things like this?